test/samples/asyncapi.json

```diff
@@ -0,0 +1,117 @@
+{
+    "asyncapid": "1.2.0",
```

Comment:
Is this typo on purpose?

Reply:
yeah, it was a forced typo to test validation. will fix it with next push